doc/rtd/index.rst

```diff
@@ -99,3 +102,11 @@ Having trouble? We would like to help!
    topics/docs.rst
    topics/testing.rst
    topics/integration_tests.rst
+.. LINKS
+.. _Code of conduct: https://ubuntu.com/community/code-of-conduct
+.. _IRC channel on Libera: https://kiwiirc.com/nextclient/irc.libera.chat/cloud-init
+.. _cloud-init mailing list: https://launchpad.net/~cloud-init
+.. _mailing list archive: https://lists.launchpad.net/cloud-init/
+.. _Contribute on Github: https://cloudinit.readthedocs.io/en/latest/topics/contributing.html
+.. _Release schedule: https://discourse.ubuntu.com/t/cloud-init-2022-release-schedule/25413
```

Comment:
Since @TheRealFalcon just posted a new release schedule, could we please update that link while we're touching this?

```diff
  diff --git a/doc/rtd/index.rst b/doc/rtd/index.rst
  index cf0614270..94d7c8821 100644
  --- a/doc/rtd/index.rst
  +++ b/doc/rtd/index.rst
  @@ -108,5 +108,5 @@ Having trouble? We would like to help!
   .. _cloud-init mailing list: https://launchpad.net/~cloud-init
   .. _mailing list archive: https://lists.launchpad.net/cloud-init/
   .. _Contribute on Github: https://cloudinit.readthedocs.io/en/latest/topics/contributing.html
  -.. _Release schedule: https://discourse.ubuntu.com/t/cloud-init-2022-release-schedule/25413
  +.. _Release schedule: https://discourse.ubuntu.com/t/cloud-init-release-schedule/32244
   .. _Report bugs on Launchpad: https://bugs.launchpad.net/cloud-init/+filebug

```